holoviews/element/chart.py

```diff
@@ -344,3 +344,23 @@ def __init__(self, data, **params):
         if isinstance(data, list) and all(isinstance(d, np.ndarray) for d in data):
             data = np.column_stack([d.flat if d.ndim > 1 else d for d in data])
         super(VectorField, self).__init__(data, **params)
+
+
+
+class Spikes(Chart):
+    """
+    Spikes is a 1D or 2D Element, which represents vertical or
+    horizontal ticks along some dimension.  If an additional dimension
```

Comment:
To me 'ticks' makes me think about plotting... how about

"Spikes is a 1D or 2D Element, which represents a series of vertical or horizontal lines distributed along some dimension..."